src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -106,14 +109,127 @@ function TokenIcons({
   )
 }
 
-function Card({ children, testID }: { children: React.ReactNode; testID: string }) {
+function Card({
+  children,
+  testID,
+  cardStyle,
+}: {
+  children: React.ReactNode
+  testID: string
+  cardStyle?: ViewStyle
+}) {
   return (
-    <View testID={testID} style={styles.card}>
+    <View testID={testID} style={[styles.card, cardStyle]}>
       {children}
     </View>
   )
 }
 
+function EarningItemLineItem({ earnItem }: { earnItem: EarningItem }) {
+  const { t } = useTranslation()
+  const tokenInfo = useTokenInfo(earnItem.tokenId)
+  const amountInUsd = tokenInfo?.priceUsd?.multipliedBy(earnItem.amount)
+  const localCurrencyExchangeRate = useSelector(usdToLocalCurrencyRateSelector)
+  const amountInLocalCurrency = new BigNumber(localCurrencyExchangeRate ?? 0).multipliedBy(
+    amountInUsd ?? 0
+  )
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+
+  return (
+    <View testID="EarningItemLineItem" style={styles.cardLineContainer}>
+      <View style={styles.cardLineLabel}>
+        <Text numberOfLines={1} style={styles.depositAndEarningsCardLabelText}>
+          {earnItem.label}
+        </Text>
+      </View>
+      <View style={styles.flexShrink}>
+        <Text style={styles.depositAndEarningsCardValueText}>
+          {t('earnFlow.poolInfoScreen.lineItemAmountDisplay', {
+            localCurrencySymbol,
+            localCurrencyAmount: formatValueToDisplay(amountInLocalCurrency),
+            cryptoAmount: formatValueToDisplay(new BigNumber(earnItem.amount)),
+            cryptoSymbol: tokenInfo?.symbol,
+          })}
+        </Text>
+      </View>
+    </View>
+  )
+}
+
+function DepositAndEarningsCard({ earnPosition }: { earnPosition: EarnPosition }) {
+  const { t } = useTranslation()
+  const { balance } = earnPosition
+  const { earningItems, depositTokenId } = earnPosition.dataProps
+  const tokenInfo = useTokenInfo(depositTokenId)
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const localCurrencyExchangeRate = useSelector(usdToLocalCurrencyRateSelector)
+
+  // Total balance in local currency
+  const depositBalanceInUsd = tokenInfo?.priceUsd?.multipliedBy(balance)
+  const depositBalanceInLocalCurrency = new BigNumber(localCurrencyExchangeRate ?? 0).multipliedBy(
+    depositBalanceInUsd ?? 0
+  )
+  const earningItemsTokenIds = earningItems.map((item) => item.tokenId)
+  const earningItemsTokenInfo = useTokensInfo(earningItemsTokenIds)
+  const totalBalanceInLocalCurrency = depositBalanceInLocalCurrency.plus(
+    earningItems.reduce((acc, item) => {
+      if (!item.subtractFromDeposit) return acc
```

Comment:
```suggestion
      if (item.subtractFromDeposit) return acc
```
I think this should be the other way around. If `subtractFromDeposit` is set, that means that it is already included in the pool balance (probably should have called it subtractFromPoolBalance). So when getting the total Deposit & Earnings, it should be pool balance plus all earning items where this is not set (i.e. add in all external rewards).

Reply:
Fixed in 9d050848e370f4a61f5a3cc513c262ea78a7ecb9!